types/react-relay/test/react-relay-tests.tsx

```diff
@@ -408,26 +419,47 @@ export const configs = [
 ];
 
 function markNotificationAsRead(source: string, storyID: string) {
-    const variables = {
-        input: {
-            source,
-            storyID,
-        },
+    // Artifact produced by relay-compiler-language-typescript
```

Comment:
might be worth turning this into a `/** start of artifacts produced by relay-compiler-language-typescript **/` and have an end one, so that it's obvious to the next folk where the responsibilities for the types lay